packages/api/src/controllers/stream.ts

```diff
@@ -482,9 +482,7 @@ app.get("/:parentId/sessions", authorizer({}), async (req, res) => {
   if (record) {
     if (record === "true" || record === "1") {
       query.push(sql`data->>'record' = 'true'`);
-      query.push(sql`data->>'recordObjectStoreId' IS NOT NULL`);
```

Comment:
I'm not sure about the backward compatibility of this query. Did we ever set `recordObjectStoreId` when `record` was `false` 🤔

Reply:
I _think_ this was some kind of flag indicating that line 1662 ran on the stream. I'm not 100% sure though tbh